cupy/testing/_array.py

```diff
@@ -74,8 +97,10 @@ def assert_array_max_ulp(a, b, maxulp=1, dtype=None):
         cupy.asnumpy(a), cupy.asnumpy(b), maxulp=maxulp, dtype=dtype)
 
 
-def assert_array_equal(x, y, err_msg='', verbose=True, strides_check=False,
-                       **kwargs):
+def assert_array_equal(
+        actual, desired, err_msg='', verbose=True, *,
+        strict=False, strides_check=False
```

Comment:
Note for later reference: `strides_check` is not NumPy-compatible. Moving to keyword-only argument in this PR.